### PR TITLE
Customize mimetypes

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 This is the custom filetype PR I have mentioned previously. This should ultimately land upstream; I'll look at that now.

File: openedx/core/lib/django_startup.py [hunk 1]
```diff
@@ -1,11 +1,25 @@
 """
-Automatic execution of startup modules in Django apps.
+Extend startup behavior shared between LMS and CMS
 """
 
 from importlib import import_module
 from django.conf import settings
 
 
+def add_mimetypes():
```
Comment (caesar2164) [inline openedx/core/lib/django_startup.py]: do we not want to put this inside of `stanford/...` dir?

Comment (stvstnfrd) [inline openedx/core/lib/django_startup.py]: No. This code already exists upstream; I'm just putting it somewhere that both LMS and CMS can import.

Comment (caesar2164) [inline openedx/core/lib/django_startup.py]: ok, then I'm 👍

**Comment by caesar2164** [comment]
👍

**Pull request merged**